### ISSUE TITLE
Decision Review pages: Move content into Drupal

### ISSUE THREAD

**Issue by AkucherGCIO** [department-of-veterans-affairs/va.gov-team#6798]
# Decision Review pages: Move content into Drupal

This is follow on work that should happen _after_ ticket #5541 is completed. It is interrelated to other tickets to updating the DR/AMA pages (like adding left nav; determining any updates to the IA). 

__Problem description:__
The Decision Review pages currently live on GH. During benefit hub migration into the CMS in 2019, these DR pages could not be migrated because they had too many content anomalies. 

__Action:__ 
As anomalies have been identified and addressed, these pages can be moved into Drupal.

__Checklist:__
[x] Move pages into Drupal. 
[ ] Send to Jen Strickland and Confer with Jen re: send pages to SMEs

__Pages:__

-  ~1. VA decisions reviews and appeals https://www.va.gov/decision-reviews/~
- [x] 2. After you request a decision review https://www.va.gov/decision-reviews/after-you-request-review/
preview-prod.vfs.va.gov/preview?nodeId=3008
- [ ] PREVIEW LINK APPROVED FOR SME REVIEW
- [x] 3. File a Supplemental Claim with new evidence  https://www.va.gov/decision-reviews/supplemental-claim/
preview-prod.vfs.va.gov/preview?nodeId=3011

- [ ] AFTER H1 CHANGED TO "**Supplemental Claims**" PREVIEW LINK APPROVED FOR SME REVIEW

[x] 4. Request a Higher-Level Review https://www.va.gov/decision-reviews/higher-level-review/
preview-prod.vfs.va.gov/preview?nodeId=3003

- [ ] AFTER H1 CHANGED TO "**Higher-Level Reviews**" PREVIEW LINK APPROVED FOR SME REVIEW

- [x] 5. Request a Board Appeal https://www.va.gov/decision-reviews/board-appeal/
preview-prod.vfs.va.gov/preview?nodeId=3029

- [ ] AFTER H1 CHANGED TO "**Board Appeals**" LINK APPROVED FOR SME REVIEW

- [x] 6. After a Board Appeal ... https://www.va.gov/decision-reviews/board-appeal/after-board-appeal-decision/
preview-prod.vfs.va.gov/preview?nodeId=3013

- [ ] PREVIEW LINK APPROVED FOR SME REVIEW

- [x] 7. Multiple party contested claims https://www.va.gov/decision-reviews/multiple-party-claims/
preview-prod.vfs.va.gov/preview?nodeId=3030

- [ ] PREVIEW LINK APPROVED FOR SME REVIEW

- [x] 8. Fiduciary claims https://www.va.gov/decision-reviews/fiduciary-claims/
preview-prod.vfs.va.gov/preview?nodeId=3016

- [ ] PREVIEW LINK APPROVED FOR SME REVIEW

- [x] 9. Insurance claims https://www.va.gov/decision-reviews/insurance-claims/
preview-prod.vfs.va.gov/preview?nodeId=3025

- [ ] PREVIEW LINK APPROVED FOR SME REVIEW

- [x] 10. FAQs https://www.va.gov/decision-reviews/faq/
preview-prod.vfs.va.gov/preview?nodeId=3004

- [ ] PREVIEW LINK APPROVED FOR SME REVIEW

- [x] 11. Get help with... https://www.va.gov/decision-reviews/get-help-with-review-request/
preview-prod.vfs.va.gov/preview?nodeId=3009

- [ ] PREVIEW LINK APPROVED FOR SME REVIEW

- [x] 12. Board hearings with a VLJ https://www.va.gov/decision-reviews/board-appeal/veterans-law-judge-hearing/
preview-prod.vfs.va.gov/preview?nodeId=3010

- [ ] PREVIEW LINK APPROVED FOR SME REVIEW

**Comment by lwalsh617** [comment]
Checklist is now on this ticket.

**Comment by RLHecht** [comment]
@lwalsh617 Higher-level reviews is built in Drupal but the number call out is not rendering correctly. i checked it off above since it's migrated but let me know if I should only check it once it's all ready to go.

**Comment by RLHecht** [comment]
@lwalsh617 Another thing we need to keep in mind and I'm not sure what ticket it should go on is after all these decision reviews pages are built in Drupal, we need to change the links in the new pages to point to nodes rather than specific pages. It should be a step that happens before we push all the pages live.

**Comment by MickinSahni** [comment]
> @lwalsh617 Another thing we need to keep in mind and I'm not sure what ticket it should go on is after all these decision reviews pages are built in Drupal, we need to change the links in the new pages to point to nodes rather than specific pages. It should be a step that happens before we push all the pages live.

@RLHecht can you confirm whether there's a ticket for the page-to-node change on this epic? 
If not I can spin one up for next sprint.

**Comment by RLHecht** [comment]
@MickinSahni There isn't one. I was first thinking it could be a step on the launch checklist but since it affects so many pages I think you're right that it should be it's own ticket with each page as a checkbox so we know to go into each one before launch and update.

**Comment by MickinSahni** [comment]
2020-03-16 PW Standup
* Create separate ticket for switching to node id for pages not already built cc @lwalsh617

**Comment by RLHecht** [comment]
The Veterans Law Judge page is brought over but I can't put in the alert that's shown below. @ncksllvn Would you be able to help with this? I don't think it's currently in Drupal as an options. The page in Drupal is node ID 3010 ![Screen Shot 2020-03-16 at 12.06.14 PM.png](https://images.zenhubusercontent.com/5c9b7e302ac54336e2c327b8/7ac2491d-64e1-4593-a0ea-ee29e27a4941)

**Comment by ncksllvn** [comment]
I see that alert banner here - http://preview-prod.vfs.va.gov/preview?nodeId=902. I think the path is `/disability/file-an-appeal/board-of-veterans-appeals`

**Comment by MickinSahni** [comment]
2020-03-18 PW Standup
* Randi's checking off pages as they're moved into Drupal

**Comment by MickinSahni** [comment]
2020-03-19 PW Standup
- Only one we can't do is the Hub page. 
- Wrapping up last page (no. 7)

**Comment by RLHecht** [comment]
@lwalsh617 @MickinSahni All these pages are now moved over to Drupal (except for the hub landing page, which we are doing in a separate ticket). I think that means we can close this ticket.

**Comment by lwalsh617** [comment]
@RLHecht Before we close it, there's that one piece of client feedback I mentioned. On the [After a board appeal decision](https://www.va.gov/decision-reviews/board-appeal/after-board-appeal-decision/) page, we have a "Mark your calendar" alert box that incorrectly says you have one year to file a Supplemental Claim. (There's no time limit on supplemental claims.)
Fred Somers suggested we remove the alert box. Our other option is to use copy we do elsewhere on the site: 
_You can file anytime if you have new evidence, but we recommend you file within one year from the date on your decision letter. This protects the most advantageous effective date for you if your claim is successful, and you may be eligible for retroactive benefit payments._
Or would that be improper use of an alert box--because there's not really a hard deadline in question??  
Once we land on a recommendation, I'll reach out to Compensation Service for approval. 
Thanks!

**Comment by RLHecht** [comment]
@lwalsh617 I removed that alert from the After a board appeal decision so I think we should be good to close.

**Comment by MickinSahni** [comment]
2020-03-20 PW
- Pages are migrated over to Drupal, Not yet validated
- Next steps
  - check with Jen Lee about sending to SMEs for review 
  - send update to jen strickland about 6375, 6399, 6697 being resolved
- close out once jen lee gives thumbs up that we don't need to wait for SME feedback

**Comment by lwalsh617** [comment]
Hey @RLHecht, 
My IA chat with @mnorthuis today raised the need for three H1 changes I'm hoping you can make in Drupal:

- [ ] https://www.va.gov/decision-reviews/higher-level-review/
In the H1, please remove "Request a" and pluralize "Reviews"so the H1  reads: **Higher-Level Reviews**

- [ ] https://www.va.gov/decision-reviews/supplemental-claim/
In the H1, please remove "File a" as well as "with new evidence" and  pluralize "Claims"so the H1  reads: **Supplemental Claims**

- [ ] https://www.va.gov/decision-reviews/board-appeal/
In the H1, please remove "Request a" and pluralize "Appeals"so the H1  reads: **Board Appeals**

Reason being, "Request a Higher-Level Review" is a better H1 for the online form page. And for consistency, and because we are ultimately adding online forms for Supplemental Claims and Board Appeals, we want to change those now as well. Let me know if you have questions. Thanks, Randi!

**Comment by MickinSahni** [comment]
2020-03-23 PW Standup
- Laura to confirm with Jen that we don't need SME approval
- Randi to put preview links into this ticket for Jen to review

**Comment by lwalsh617** [comment]
**Confirming that you saw my message just above Mickin's ^^ regarding changing the H1 on 3 of the DR pages.** Once we make this change, we should be ready to share preview links with SMEs for a final fact check. 
 @mnorthuis -- this is per our IA discussion. Look good from your end?
@jenniferlee-dsva -- Just keeping you in the loop
@RLHecht -- Flagging here because this came up after the screenshots you sent me. 
Thanks all!

**Comment by jenniferlee-dsva** [comment]
@lwalsh617 @RLHecht - I've reviewed the preview pages above and am uploading comments here as a Word doc. Mostly regarding H sizes and semantic ordering/content hierarchy. Happy to jump on a call if anything needs clarification. 

Dang it - still can't upload to GH. Will Slack you doc. 

REplying to @MickinSahni ^ re SME approval - As long as these follow the approaches that @lwalsh617 briefed the SMEs on with her IA and copydeck presentations, these can be sent to them as FYSA rather than to collect another round of feedback. (But defer to @lwalsh617 based on her latest convos with SMEs.) 

I recommend noting that they can preview and flag anything _**factually**_ incorrect, and if we don't hear from them by X date, these will go live with the daily deploy. I wouldn't give them too much time - this should be a quick read through for factual errors, since the business already approved/vetted these the first time these pages went live.

**Comment by MickinSahni** [comment]
2020-03-25 PW Standup
- Work for this ticket is complete and we have a follow-on ticket in #6922 for Sprint 18. 
- We're able to close this out
